[BUG] GHA workflow fails "Building static executable images is only supported with musl libc"

**Describe the bug**

```
Run native-image --enable-url-protocols=https --static -H:+ReportExceptionStackTraces -jar astra-cli-*.jar astra-cli-linux
  native-image --enable-url-protocols=https --static -H:+ReportExceptionStackTraces -jar astra-cli-*.jar astra-cli-linux
  shell: /usr/bin/bash -e {0}
  env:
    GRAALVM_HOME: /opt/hostedtoolcache/graalvm-jdk-[2](https://github.com/datastax/astra-cli/actions/runs/12673156166/job/35318906565#step:4:2)1_linux-x64_bin/21.0.0/x64/graalvm-jdk-21.0.5+9.1
    JAVA_HOME: /opt/hostedtoolcache/graalvm-jdk-21_linux-x64_bin/21.0.0/x64/graalvm-jdk-21.0.5+9.1
========================================================================================================================
GraalVM Native Image: Generating 'astra-cli-linux' (static executable)...
========================================================================================================================
For detailed information and explanations on the build output, visit:
https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
------------------------------------------------------------------------------------------------------------------------
[1/8] Initializing...                                                                                    (6.6s @ 0.11GB)
 Java version: 21.0.5+9-LTS, vendor version: Oracle GraalVM 21.0.5+9.1
 Graal compiler: optimization level: 2, target machine: x86-64-v[3](https://github.com/datastax/astra-cli/actions/runs/12673156166/job/35318906565#step:4:3), PGO: ML-inferred
 C compiler: gcc (linux, x86_6[4](https://github.com/datastax/astra-cli/actions/runs/12673156166/job/35318906565#step:4:4), 11.4.0)
 Garbage collector: Serial GC (max heap size: 80% of RAM)
 1 user-specific feature(s):
 - com.oracle.svm.thirdparty.gson.GsonFeature
------------------------------------------------------------------------------------------------------------------------
Build resources:
 - 10.93GB of memory (70.0% of 1[5](https://github.com/datastax/astra-cli/actions/runs/12673156166/job/35318906565#step:4:5).61GB system memory, determined at start)
 - 4 thread(s) (100.0% of 4 available processor(s), determined at start)

Error: Building static executable images is only supported with musl libc. Remove the '--static' option or add the '--libc=musl' option.
com.oracle.svm.core.util.UserError$UserException: Building static executable images is only supported with musl libc. Remove the '--static' option or add the '--libc=musl' option.
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.UserError.abort(UserError.java:73)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.SubstrateOptions.lambda$static$0(SubstrateOptions.java:119)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.option.HostedOptionKey.validate(HostedOptionKey.java:107)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.option.ValidateImageBuildOptionsFeature.validate(ValidateImageBuildOptionsFeature.java:52)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.option.ValidateImageBuildOptionsFeature.beforeAnalysis(ValidateImageBuildOptionsFeature.java:44)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.lambda$runPointsToAnalysis$9(NativeImageGenerator.java:773)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.FeatureHandler.forEachFeature(FeatureHandler.java:90)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:773)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:592)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:550)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:539)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:721)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.start(NativeImageGeneratorRunner.java:143)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:98)
------------------------------------------------------------------------------------------------------------------------
                        0.3s (4.4% of total time) in 15 GCs | Peak RSS: 0.51GB | CPU load: 2.22
========================================================================================================================
Finished generating 'astra-cli-linux' in [6](https://github.com/datastax/astra-cli/actions/runs/12673156166/job/35318906565#step:4:6).7s.
Error: Process completed with exit code 1.
```

**To Reproduce**

Steps to reproduce the behavior:

See https://github.com/datastax/astra-cli/actions/runs/12673156166/job/35318906565

**Expected behavior**
GHA workflow successful

**Screenshots**
n/a

**Desktop (please complete the following information):**
- OS: [e.g. MacOS]
- Version [e.g. 0.1.XXX]

**Additional context**
n/a

--- a/.github/workflows/cli-native-images.yml
+++ b/.github/workflows/cli-native-images.yml
@@ -59,7 +59,7 @@ jobs:
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Build native image on Linux
-        run: native-image --enable-url-protocols=https --static -H:+ReportExceptionStackTraces -jar astra-cli-*.jar astra-cli-linux
+        run: native-image --enable-url-protocols=https -H:+ReportExceptionStackTraces -jar astra-cli-*.jar astra-cli-linux
         if: runner.os == 'Linux'
       - name: Build native image on Mac OS X
         run: native-image --enable-url-protocols=https -H:+ReportExceptionStackTraces -jar astra-cli-*.jar astra-cli-macos

